routes/benine.js

```diff
@@ -0,0 +1,48 @@
+import express from "express";
+import BenineHelper from "../objects/BenineHelper";
+import CameraShot from "../objects/CameraShot";
+import ProjectManager from "../objects/ProjectManager";
+
+const router = express.Router(); // eslint-disable-line new-cap
+
+// Get users data
+router.get("/get-cameras", (req, res) => {
+    const benineHelper = new BenineHelper();
+    benineHelper.getCameras((cameras) => {
+        res.json({ cameras });
+    });
+});
+
+// Get presets data from benine
+router.get("/get-presets", (req, res) => {
+    const benineHelper = new BenineHelper();
+    benineHelper.getPresets((presets) => {
+        res.json({ presets });
+    });
+});
+
+// Get presets data from benine for a certain camera
+router.get("/get-presets/:id(\\d+)", (req, res) => {
```

Comment:
Is this id the id of the camera? If so, please change the parameter name to `cameraId`. It looks like the current format gets the preset according to the id of the preset you are getting.